dev/tools/controllerbuilder/pkg/commands/generatetypes/generatetypescommand.go

```diff
@@ -185,7 +185,8 @@ func RunGenerateCRD(ctx context.Context, o *GenerateCRDOptions) error {
 	}
 
 	addCopyright := true
-	if err := typeGenerator.WriteFiles(addCopyright); err != nil {
+	writeEmptyFiles := true
```

Comment:
I may miss something. But this looks like we can just not call the `typeGenerator.WriteFiles` if wirteEmptyFiles is true, right?

Reply:
Not sure I follow: we want to write the empty file